javascript/src/events/event-reporter.ts

```diff
@@ -35,47 +39,12 @@ export class EventReporter {
       event,
     });
 
-    if (!this.endpoint) {
+    if (!this.eventsEndpoint) {
       this.logger.warn(
         "No LANGWATCH_ENDPOINT configured, skipping event posting"
       );
       return;
     }
 
-    try {
-      const response = await fetch(`${this.endpoint}/api/scenario-events`, {
```

Comment:
Did you delete the whole fetching mechanism?

Reply:
@0xdeafcafe

Reply:
Nope, restored here https://github.com/langwatch/scenario/pull/19/commits/83dd5e5e05b301f6ed3e5efd329e1f34cd601dff